pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -240,6 +247,25 @@ func (c *Cluster) enablePrometheusModule(clusterName string) error {
 	return nil
 }
 
+func (c *Cluster) enableMgrModules(context *clusterd.Context, clusterName string, mgrSpec cephv1.MgrSpec) error {
```

Comment:
If modules are removed from the CR, do we need to disable them? I'm ok with just enabling them if they're in the list, just wanted to confirm if that was the intention.

Reply:
If they get removed, nothing happens, they will remain enabled. That's the same issue has ConfigOverrideName with deletion.